sihl/test/database.ml

```diff
@@ -162,6 +162,35 @@ let choose_database_pool _ () =
   Lwt.return ()
 ;;
 
+let drop_database_pool _ () =
+  let open Sihl.Database in
+  let%lwt () = fetch_pool () |> Caqti_lwt.Pool.drain in
+  let database_url =
+    Option.value
+      ~default:"not found"
+      (Sihl.Configuration.read_string "DATABASE_URL")
+  in
+  let label = "foo" in
```

Comment:
@mabiede where does "foo" come from? is it just a placeholder?

Reply:
@joseferben It's a dummy database label, which is also used in the previous test.